apps/examples/testcase/ta_tc/task_manager/utc/utc_task_manager_main.c

```diff
@@ -58,8 +60,28 @@ static int broad_wifi_off_cnt;
 
 static sem_t tm_broad_sem;
 
+static void sync_test_cb(tm_unicast_msg_t *info)
+{
+	tm_unicast_msg_t reply_msg;
+
+	if (strncmp(info->msg, TM_SYNC_SEND_MSG, info->msg_size) == 0) {
+		reply_msg.msg_size = strlen(TM_SYNC_RECV_MSG);
```

Comment:
I think this line should be in below if conditional.
Because when allocation of reply_msg is failed, size should be zero, not strlen(TM_SYNC_RECV_MSG).